SabreTools.DatTools/Rebuilder.cs

```diff
@@ -182,6 +189,9 @@ public class Rebuilder
             bool date = false,
             bool delete = false,
             bool inverse = false,
+            bool baseReplace = false,
+            bool archivesAsFiles = false,
+            string symlinkDir = "",           
```

Comment:
You need to add these to the list of parameters. Please use `string.Empty` instead of `""`.